$NetBSD: patch-ab,v 1.4 2018/01/02 21:11:10 rillig Exp $

C90 compilers do not accept declarations after the first statement.

Xbae-4.60.4 doesn't need this patch anymore since the code looks entirely
different.

--- src/Matrix.c.orig	Wed Dec  8 20:11:39 2004
+++ src/Matrix.c
@@ -3023,6 +3023,7 @@ static void TraverseInTimeOut(XtPointer 
  */
 static void TraverseInCB(Widget clip, XbaeMatrixWidget mw, XtPointer call_data)
 {
+        Boolean mapped;
         DEBUGOUT(_XbaeDebug2(__FILE__, (Widget) mw, clip, "TraverseInCB\n"));
 
         /*
@@ -3031,7 +3032,6 @@ static void TraverseInCB(Widget clip, Xb
          * are disallowed in 1.2 (we may be in this CB as a result of someone
          * calling XmProcessTraversal).  So we call it through a zero length timeout
          */
-        Boolean mapped;
         XtVaGetValues(TextChild(mw), XmNmappedWhenManaged, &mapped, NULL);
 
         if (mapped) {
